rusk/src/bin/config/chain.rs

```diff
@@ -96,6 +97,10 @@ impl ChainConfig {
         self.min_deploy_points
     }
 
+    pub(crate) fn public_sender_start_height(&self) -> Option<u64> {
```

Comment:
How about calling it something with `activation`?
```suggestion
    pub(crate) fn public_sender_activation_height(&self) -> Option<u64> {
```
or if we expect more changes like this in the future:
```suggestion
    pub(crate) fn activation_height_public_sender(&self) -> Option<u64> {
```

Reply:
Indeed we can expect more changes like that in the future. 
Maybe a dedicated struct would be a better fit

Maybe something like this?

```toml
[chain.features]
abi_sender = activaction_height
feature_x = activaction_height_x
feature_y = activaction_height_y
```

Reply:
I added the dedicated section `[vm.features]`